src/protocol/recordBatch/v0/index.js

```diff
@@ -4,6 +4,9 @@ const crc32C = require('../crc32C')
 const { Types: Compression, lookupCodec } = require('../../message/compression')
 
 const MAGIC_BYTE = 2
+const COMPRESSION_MASK = 11 // The lowest 3 bits
```

Comment:
Why not continue to use `3`, as we have in the [compression module](https://github.com/tulios/kafkajs/blob/master/src/protocol/message/compression/index.js#L3)? As far as I can tell they're effectively the same, except this masks a 4th bit that won't be present.

Reply:
You are right, I tried to move away from the hex definition but messed up the mask. It should be `3`. We will have to bump this in the future to support ZSTD, but let's do it when the time comes.

Reply:
ef252f7